src/pages/components/logo-grid.mdx

```diff
@@ -23,9 +24,19 @@ Logo grid consists of a heading and a group of logos. Each logo should be contai
 
 ![default](../..//images/pattern/logo-grid/logo-grid.jpg)
 
+## Content guidance
+
+| Field | Field type | Required   | Cardinality | Max character size (English / translated) |
+| -------- | --------- | ---------- | ---------- | ----------|
+| Headline | Short copy | No | 1 | S (75 / 100)|
```

Comment:
I believe Logo Grid uses Content block's headline and CTA here. Just calling that out.

Reply:
Yeah I didn't use it just to make it a little simpler.

Reply:
As I mentioned in another PR it might be nice to align on a rule per say on when we link and when we don't. It's a little unclear to me at the moment which is why I keep calling it out.